ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/port_channel_interfaces.py

```diff
@@ -174,3 +176,32 @@ def _get_port_channel_subinterface_cfg(self, subinterface: dict, adapter: dict,
             }
 
         return strip_null_from_data(port_channel_interface)
+
+    def _check_duplicate(self, candidate_port_channel_config, port_channel_interfaces) -> None:
```

Comment:
```suggestion
    def _add_if_not_duplicate(self, candidate_port_channel_config, port_channel_interfaces) -> None:
```
or something similar

Reply:
Addressed in [b748a7c](https://github.com/aristanetworks/ansible-avd/pull/2651/commits/b748a7c018ca064194601d05ff609b5550cb9412)